openapi-codegen/sbt-plugin/src/sbt-test/sbt-openapi-codegen/oneOf-json-roundtrip_jsoniter_scala3/src/test/scala/JsonRoundtrip.scala

```diff
@@ -142,6 +148,23 @@ class JsonRoundtrip extends AnyFreeSpec with Matchers {
         1.second
       )
     }
+  }
 
+  "schemas generated for recursive types are ok" in {
```

Comment:
In a sense, this test is redundant, since the `scripted` tests will just timeout on the `run` stage without the fix. But tests are still nice, aren't they?